### PR TITLE
feat: added rating component

### PR DESCRIPTION
**Pull request by Asuraking1n**
added rating component

### CODE REVIEW

File: css/rating.css [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    align-items: center;
+    border: 3px solid var(--primary--color);
+
+}
+
+
+*{
+    margin: 0;
+    padding: 0;
+}
```
Comment (vikrantkode) [inline css/rating.css:24]: this you can add inside the root element which you are importing

File: css/rating.css [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    cursor:pointer;
+    font-size:30px;
+    color:#ccc;
+}
+.rate:not(:checked) > label:before {
+    content: '★ ';
+}
+.rate > input:checked ~ label {
+    color: #ffc700;    
+}
```
Comment (vikrantkode) [inline css/rating.css:48]: use variable to add style color

**Review by vikrantkode — COMMENTED** [review]
Overall looks good to me.
Some minor changes suggested which you can look when refactoring.

**Comment by Asuraking1n** [comment]
> Overall looks good to me. Some minor changes suggested which you can look when refactoring.

Sure...I will correct them

**Review by arulk8 — COMMENTED** [review]
Looks good for me

**Pull request merged**